Lexplorer/Pages/NFTDetail.razor

```diff
@@ -47,6 +47,18 @@
             <td>Animation URL</td>
             <td>@nftMetadata?.animation_url</td>
         </tr>
+        @if(nftMetadata?.attributes != null && nftMetadata?.attributes!.Count > 0)
```

Comment:
Similar to last time this can be simplified to `nftMetadata?.count ?? 0 > 0`. 

?? means, use right part, if left part is null.

Reply:
im having trouble adding this into the if statement. can you show me how that would look like or commit it yoursel?

Reply:
Sorry, I should have tried myself. Just committed the even simpler form, since Count is a non-nullable int.